h2o-docs/src/product/grid-search.rst

```diff
@@ -42,6 +42,8 @@ Grid Search in R and Python
           parameters (e.g., ``list(ntrees = c(1,100), learn_rate = c(0.1, 0.001))``)
        -  ``search_criteria`` optional attribute for specifying a more
           advanced search strategy  
+       -  ``parallelism`` (number of models built in parallel) allows 
```

Comment:
Add more info for both R and Python

``parallelism`` The number of models to build in parallel. Parallelism allows the leader node to search the hyperspace and build models in a parallel way, which ultimately speeds up grid search on small data. A value of 1 (default) specifies sequential building. Specify 0 for adaptive parallelism, which is decided by H2O. Any number > 1 sets the exact number of models built in parallel.